docs/user/custom-domains.rst

```diff
@@ -1,126 +1,85 @@
 Custom Domains
 ==============
 
-Custom domains allow you to serve your documentation from your own domain.
-This is great for maintaining a consistent brand for your documentation and application.
+You can serve your documentation project from your own domain,
+for instance ``docs.example.com``.
+This is great for maintaining a consistent brand for your product and its documentation.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+*By default*, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
-For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
+.. seealso::
 
-.. contents:: Contents
-    :local:
+    :doc:`/guides/custom-domains`
+        Information on creating and managing custom domains,
+        and common configurations you might use to set up your domain
 
-Adding a custom domain
-----------------------
+How custom domains work
+-----------------------
 
-To setup your custom domain, follow these steps:
+To use a custom domain, you enter the domain in your Read the Docs project's :guilabel:`Admin` and update your DNS provider with a new DNS entry.
 
-#. Go the :guilabel:`Admin` tab of your project.
-#. Click on :guilabel:`Domains`.
-#. Enter the domain where you want to serve the documentation from (e.g. ``docs.example.com``).
-#. Mark the :guilabel:`Canonical` option if you want use this domain
-   as your :doc:`canonical domain </canonical-urls>`.
-#. Click on :guilabel:`Add`.
-#. At the top of the next page you'll find the value of the DNS record that you need to point your domain to.
-   For |org_brand| this is ``readthedocs.io``, and for :doc:`/commercial/index`
-   the record is in the form of ``<hash>.domains.readthedocs.com``.
+These two actions are all that are needed from you.
+Once the DNS record has propagated,
+Read the Docs automatically issues an SSL certificate through Cloudflare and starts serving your documentation.
 
-   .. note::
+.. mermaid::
 
-      For a subdomain like ``docs.example.com`` add a CNAME record,
-      and for a root domain like ``example.com`` use an ANAME or ALIAS record.
+    graph TD
+        subgraph rtd [On Read the Docs]
+          A(fa:fa-pencil Add docs.example.com as Custom domain)
+          A-->AA(fa:fa-spinner Your project is rebuilt with<br>docs.example.com as a canonical domain)
```

Comment:
This won't happen automatically, will it? They need to rebuild the project after the canonical domain is properly configured.

I think this should be moved to after the `Wait for DNS` step, probably at the end? 

Something like `Next time your project is built, it will update the :doc:/canonical-urls to use docs.example.com`